service/src/main/java/bio/terra/drshub/controllers/GlobalExceptionHandler.java

```diff
@@ -55,7 +55,7 @@ public ResponseEntity<ErrorReport> validationExceptionHandler(Exception ex) {
   @ExceptionHandler(HttpStatusCodeException.class)
   public ResponseEntity<ErrorReport> HttpStatusCodeExceptionReportHandler(
       HttpStatusCodeException ex) {
-    return buildErrorReport(ex, ex.getStatusCode());
+    return buildErrorReport(ex, HttpStatus.valueOf(ex.getStatusCode().value()));
```

Comment:
I think a better fix for this is to change `buildErrorReport()` to accept `HttpStatusCode` instead of `HttpStatus` (since `HttpStatusCode` is a subclass of `HttpStatus`, and `ex.getStatusCode()` returns a `HttpStatusCode`).
```
    return buildErrorReport(ex, getStatusCode());
...
  private ResponseEntity<ErrorReport> buildErrorReport(
      @NotNull Throwable ex, HttpStatusCode statusCode) {
```